core/src/main/java/org/geysermc/geyser/event/type/GeyserBedrockPingEventImpl.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2019-2022 GeyserMC. http://geysermc.org
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/Geyser
+ */
+
+package org.geysermc.geyser.event.type;
+
+import com.nukkitx.protocol.bedrock.BedrockPong;
+import org.geysermc.geyser.api.event.lifecycle.GeyserBedrockPingEvent;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.net.InetSocketAddress;
+import java.util.Objects;
+
+public class GeyserBedrockPingEventImpl implements GeyserBedrockPingEvent {
+    private final InetSocketAddress address;
+    private final BedrockPong pong;
+
+    public GeyserBedrockPingEventImpl(BedrockPong pong, InetSocketAddress address) {
+        this.address = address;
+        this.pong = pong;
+    }
+
+    @Override
+    public void setMotd1(@Nonnull String motd1) {
+        Objects.requireNonNull(motd1, "Motd1 cannot be null");
+        pong.setMotd(motd1);
+    }
+
+    @Override
+    public void setMotd2(@Nonnull String motd2) {
+        Objects.requireNonNull(motd2, "Motd2 cannot be null");
+        pong.setSubMotd(motd2);
+    }
+
+    @Override
+    public void setPlayerCount(int count) {
+        if (count <= -1) throw new IllegalArgumentException("Player count cannot be below 0");
```

Comment:
I don't mean to be picky but just do `count < 0` instead - more readable/understandable